maven-failsafe-plugin/src/main/java/org/apache/maven/plugin/failsafe/VerifyMojo.java

```diff
@@ -148,10 +148,9 @@ public class VerifyMojo extends AbstractMojo implements SurefireReportParameters
     private int failOnFlakeCount;
 
     /**
-     * The character encoding scheme to be applied.
-     * Deprecated since 2.20.1 and used encoding UTF-8 in <code>failsafe-summary.xml</code>.
+     * No effect. UTF-8 is always used for <code>failsafe-summary.xml</code>.
      *
-     * @deprecated since of 2.20.1
+     * @deprecated since 2.20.1
```

Comment:
Question: As it is deprecated since 2.20 with current version 3.x and new major version coming with Maven 4. Are there plans to remove this property completely? 

-----
The doc updates gets a non-binding approval by me.